packages/client/src/controllers/engine.ts

```diff
@@ -638,10 +672,8 @@ export class Engine extends IEngine {
   ) => {
     // TODO(ilja) validation
     const { id } = payload;
-    await this.client.core.relayer.unsubscribe(topic);
     await this.sendResult<"wc_sessionDelete">(id, topic, true);
-    await this.client.session.delete(topic, ERROR.DELETED.format());
-    await this.client.core.crypto.deleteSymKey(topic);
```

Comment:
`this.deleteSession` now calls `this.client.core.crypto.deleteKeyPair(topic)` internally instead of `this.client.core.crypto.deleteSymKey(topic)` as shown here previously. 

Is that intentional?

Reply:
Yes, however this is to be confirmed with @pedrouid Before deleting session and pairing called `deleteSymKey` for both, but I believe it needs to be `deleteKeyPair` for session and `deleteSymKey` for pairing

Reply:
you always need to `deleteSymKey(topic)` for both pairing and session

however for session you also have a keypair mapped by publicKey under `self` which needs to be deleted with `deleteKeyPair(publicKey)`